packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -551,8 +551,8 @@ function LineItemSublabelAndPrice( {
 			return (
 				<>
 					{ translate(
-						'Monthly subscription: %(itemPrice)s x %(members)s active member',
-						'Monthly subscription: %(itemPrice)s x %(members)s active members',
+						'Monthly subscription: %(itemPrice)s x %(members)s member',
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.